test/Microsoft.AspNet.Authentication.Test/Cookies/CookieMiddlewareTests.cs

```diff
@@ -1023,14 +1023,9 @@ private static void Describe(HttpResponse res, AuthenticateContext result)
             {
                 xml.Add(result.Properties.Select(extra => new XElement("extra", new XAttribute("type", extra.Key), new XAttribute("value", extra.Value))));
             }
-            using (var memory = new MemoryStream())
-            {
-                using (var writer = new XmlTextWriter(memory, Encoding.UTF8))
-                {
-                    xml.WriteTo(writer);
-                }
-                res.Body.Write(memory.ToArray(), 0, memory.ToArray().Length);
-            }
+
+            var xmlBytes = Encoding.UTF8.GetBytes(xml.ToString());
```

Comment:
This looks fine to me. It strikes me as something where either it totally works or it totally doesn't...